.github/actions/spell-check/patterns/patterns.txt

```diff
@@ -7,3 +7,4 @@ Scro\&ll
  :\\windows\\syste\b
 TestUtils::VerifyExpectedString\(tb, L"[^"]+"
 hostSm\.ProcessString\(L"[^"]+"
+([A-Za-z])\1{3,}
```

Comment:
This seems to work reasonably well in this repository.

For powerdns/pdns i'm going to add `\b` around this line.

Otherwise, a `aaarec` which is a typo would be accepted along w/ `aaaarec` which is not a typo.

Reply:
Oh, that's actually a good call. This should probably be book-ended in `\b`s so it only hits full words and breaks on boundaries.